src/dataflow-types/src/client/controller.rs

```diff
@@ -156,7 +155,7 @@ pub struct Controller<T = mz_repr::Timestamp> {
 
 impl<T> Controller<T>
 where
-    T: Timestamp + Lattice + Codec64 + Copy + Unpin,
+    T: Timestamp + Lattice + Codec64 + Copy + Unpin + ProtoSerializableTimestamp,
```

Comment:
I think its a bit weird to have this trait implement on the `Timestamp`, I think this works:

```suggestion
    T: Timestamp + Lattice + Codec64 + Copy + Unpin,
    ComputeCommand<T>: RustType<ProtoComputeCommand>,
    ComputeResponse<T>: RustType<ProtoComputeResponse>,
```

don't block on me, but we should definitely clean this up after!

Reply:
Oh this is great, and totally works!